Loops/Fix infinite execution/task.md

```diff
@@ -0,0 +1,24 @@
+## Fix infinite execution
+
+In the code editor, there is a `while` loop, defined inside a [function](course://Functions/Definition). Please do not worry about it at the moment,
```

Comment:
In the code editor, there is a `while` loop defined inside a [function](course://Functions/Definition). Please do not worry about it at the moment,